### ISSUE TITLE
ETA (Estimated Time of Arrival) calculation.

### ISSUE THREAD

**Issue by umitkose** [graphhopper/jsprit#518]
I am trying to calculate ETA for a list of delivery jobs.
After getting a solution by following the path  Solution > Route> TourActivity we can access getArrTime and getEndTime methods. 
Can we calculate the ETA of a delivery by startTime + getArrTime?  I tried this formula but I guess getArrTime only respects to service durations assinged to delivery job.

If someone confirm my formula I will search my implementation to where I made a mistake.

Thanks in advance.

**Comment by bludginozzie** [comment]
Yes, I think your formula is correct.  In essence the `TourActivity.getArrTime()` can be considered an ETA for an activity.
